content/en/api_catalog/monitor_apis.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Monitoring your APIs
+title: Monitoring APIs
```

Comment:
I think this one should hold only the last three titles ("Alerting on endpoints..", "Tracking and improving..", "Finding and closing...")